supervisor/shared/filesystem.c

```diff
@@ -69,6 +69,18 @@ static void make_empty_file(FATFS *fatfs, const char *path) {
     f_close(&fp);
 }
 
+
+static void make_sample_code_file(FATFS *fatfs) {
+    FIL fs;
+    UINT *char_written = 0;
+    const byte buffer[] = "print('Hello World!')";
+
+    //Create or modify existing code.py file 
+    f_open(fatfs, &fs, "/code.py", FA_WRITE | FA_CREATE_ALWAYS);
+    f_write(&fs, buffer, sizeof(buffer), char_written);
```

Comment:
This will
 * include a possibly-invisible "\0" character at the end of the file, because` "abc"` is the same as the character array `{'a', 'b', 'c, '\0'}` with a size of 4 bytes.
 * not include a newline character at the end of the file.  Technically, a text file has each line end with a newline character
 * (I don't know if we want to ship CRLF files for people suffering on Windows systems, or whether Windows now gracefully handles UNIX style text files)

Recommendation: add `\n` to the end of the buffer[], and subtract 1 from sizeof(buffer) at f_write

Reply:
Totally,  makes sense ! :)